app/src/organisms/GripperWizardFlows/MovePin.tsx

```diff
@@ -205,7 +207,14 @@ export const MovePin = (props: MovePinProps): JSX.Element | null => {
         alternativeSpinner={inProgressImage}
       />
     )
-  return (
+  return errorMessage != null ? (
+    <SimpleWizardBody
+      isSuccess={false}
+      iconColor={COLORS.errorEnabled}
+      header={t('error_encountered')}
```

Comment:
https://github.com/Opentrons/opentrons/assets/474225/ccb7480c-3fe7-42a0-bdd9-37be21310427

I think moving error_encountered into `shared.json` would be great.

```suggestion
      header={t('shared:error_encountered')}
```

or just add  pipette_wizard_flows to `useTranslation`
```suggestion
      header={t('pipette_wizard_flows:error_encountered')}
```